### ISSUE TITLE
[Bug] Windows Search Indexer high CPU

### ISSUE THREAD

**Issue by talkshowhost** [ScoopInstaller/Scoop#4920]
#### Current Behavior
My Windows search indexer was constantly having a high CPU usage, trying to index many files. Looking on my Windows settings I had over 700k files indexed and 100k pending, which compared to other pcs not using scoop they had between 10-20k. 

Investigating further I noticed scoop seems to already add excluded indexing folders by default, like the ones in `/buckets`. The problem was that most of the files in my scenario come from the `/apps` and `/persist` folders. 

This was probably worsened by having apps like aws, nodejs and nodejs-lts which have theis caches under these folders with thousands of files.

#### Expected Behavior
Windows search indexer shouldn't be slowed down by scoop. Probably not all those files need to be indexed.

#### Possible Solution
Exclude `/apps` and `/persist` folders by default, since `/shims` should be enough for most use cases to access the installed apps through scoop.

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** 5.1.22000.652

#### Scoop Configuration

```json
{
    "lastupdate":  "2022-05-09T13:34:21.1295483+01:00",
    "SCOOP_REPO":  "https://github.com/lukesampson/scoop",
    "SCOOP_BRANCH":  "master"
}
```

**Comment by talkshowhost** [comment]
Fair enough, I don't know if this is happening only to me and if so why, but maybe I would suggest to add this in the FAQ for other people if they come across this.

Thanks!

**Comment by rashil2000** [comment]
I have personally not faced this problem, however what I have observed is that the Indexer has some heuristics to detect code folders (repositories), or in general any folders with a large number of small files - and adds exclusions for them automatically.
